doc(release) Release notes web 24.10.6 (#4244)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-24.10/releases/centreon-os.mdx

## Centreon Web

### 24.10.6

Release date: `March 31, 2025`

<details open>
  <summary>Enhancements</summary>

- [Resource Status] Optimized search query.

> - [CMA] Centreon Monitoring Agent Connector is now created during install/upgrade.

Reviewer: ```suggestion
- [CMA] The Centreon Monitoring Agent connector is now automatically created during install/upgrade.
```

</details>

<details>
  <summary>Bug fixes</summary>

> - [Configuration] Fixed ACL check on the contact / user configuration page.

Reviewer: ```suggestion
- [Configuration] Fixed ACL checks on contact/user configuration page.
```

> - [Configuration] Fixed an issue on notification period field which was not saved in some cases.

Reviewer: ```suggestion
- [Configuration] Fixed an issue where the notification period field was not saved in some cases.
```

> - [Custom Views] Fixed issue preventing non admin users from seeing hostgroups in the Hostgroup Monitoring widget.

Reviewer: ```suggestion
- [Custom Views] Fixed an issue preventing non admin users from seeing host groups in the Host group Monitoring widget and service groups in the Service group Monitoring widget.
```

> - [Custom Views] Fixed issue preventing non admin users from seeing servicegroups in the Servicegroup Monitoring widget.

Reviewer: ```suggestion
```

> - [Dashboards] Fixed issue forcing edit mode on dashboards containing a Resource Table widget.

Reviewer: ```suggestion
- [Dashboards] Fixed an issue where dashboards containing a Resource Table widget appeared in Edit mode.
```

> - [Dashboards][Metrics Graph] Optimized bar chart rendering when modifying opacity or bar radius.

Reviewer: ```suggestion
- [Dashboards][Metrics Graph] Optimized bar chart rendering performance when modifying opacity or bar radius.
```

> - [Discovery] Fixed autodiscovery credentials lost during update.

Reviewer: ```suggestion
- [Discovery] Fixed an issue where autodiscovery credentials were lost during an update.
```

> - [Downtimes] Fixed an issue on displaying the downtimes pages for users under ACLs.

Reviewer: ```suggestion
- [Downtimes] Fixed an issue where non admin users could not display the downtimes page.
```

> - [Downtimes] Fixed an issue where loading downtimes page resulted in gateway timeout error.

Reviewer: ```suggestion
- [Downtimes] Fixed an issue where loading the downtimes page resulted in a gateway timeout error.
```

> - [Graphs] Fixed issue preventing non admin users from exporting graphs in ".png".

Reviewer: ```suggestion
- [Graphs] Fixed an issue preventing non admin users from exporting graphs as ".png".
```

> - [Resource Status] Fixed an issue on Host & Services columns that doesn't contain the rights infos in the view by host.

Reviewer: ```suggestion
- [Resource Status] Fixed an issue where the Host & Service columns didn't contain the right information when displayed in view by host mode.
```

> - [Service Groups] Fixed issue preventing non admin users from displaying Service Groups.

Reviewer: ```suggestion
- [Service Groups] Fixed an issue preventing non admin users from displaying Service Groups.
```

- [Translation]	Fixed a translation issue on menu & breadcrumb and added a "Connectors" submenu that includes Monitoring Connectors and Additional configurations.

> - [Web] Fixed issue preventing users of some password managers from accessing to legacy pages.

Reviewer: ```suggestion
- [Web] Fixed an issue where some password managers were preventing users from accessing legacy pages.
```
